.gitignore

```diff
@@ -210,4 +213,8 @@ dmypy.json
 .vscode
 
 # keep staticfiles directory
-!project/staticfiles/.gitkeep
\ No newline at end of file
+!project/staticfiles/.gitkeep
+<<<<<<< HEAD
+!project/media/.gitkeep
+=======
+>>>>>>> 6cbece5ae023814e69a4ef3868ff8e0691e5fcde
```

Comment:
Apparently, you somehow include commit numbers in your commit. Could you remove them and make the file cleaner?